### DISCUSSION TITLE
Welcome to gkd Discussions!

### DISCUSSION THREAD

**Discussion by lisonge** [gkd-kit/gkd#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lingzeyang** [comment]
gkd挂不住后台,作个对比:李跳跳后台比gkd强

**Reply by lisonge** [reply]
可以看看这个说明 <https://github.com/gkd-kit/gkd/discussions/43?converting=1#discussioncomment-7042413>

**Comment by ztszts777** [comment]
请求加入每日英语听力的开屏广告屏蔽

**Reply by stone0118** [reply]
可以自己手动加一下哦

**Comment by ztszts777** [comment]
gspace无法跳过广告，设备华为p60

**Comment by lisonge** [comment]
@ztszts777 

https://github.com/gkd-kit/subscription/issues/new/choose

**Comment by zhang-0318** [comment]
请求加入 乔安智联 的开屏广告屏蔽

**Reply by lisonge** [reply]
https://github.com/gkd-kit/subscription/issues
